pkg/manager/member/tikv_member_manager.go

```diff
@@ -410,6 +411,10 @@ func getNewTiKVSetForTidbCluster(tc *v1alpha1.TidbCluster) (*apps.StatefulSet, e
 	if tc.Spec.TiKV.HostNetwork {
 		dnsPolicy = corev1.DNSClusterFirstWithHostNet
 	}
+	tz := tc.Spec.Timezone
+	if tz != "UTC" {
```

Comment:
```suggestion
	if tz != "UTC" && tz != "" {
```
Shall we check for empty string?